source/_integrations/zwave_js.markdown

```diff
@@ -124,6 +124,38 @@ While your Z-Wave mesh is permanently stored on your dongle, the additional meta
+
+### Integration configuration panel
+
+The following features can be accessed from the integration configuration panel:
+
+![Z-Wave integration configuration panel](/images/integrations/z-wave/z-wave-integration-config-panel.png)
+
+* **Add device:** Allows you to pre-provision a SmartStart device or start the inclusion process for adding a new device to your network.
+* **Remove device:** Starts the exclusion process for removing a device from your network.
+* **Heal network:** Forces your network to rediscover routes to the controller from each device. This is useful when devices or the controller have moved to a new location, or if you are having significant problems with your network, but it also generates a lot of network traffic and should be used sparingly.
+* **[Controller statistics](https://zwave-js.github.io/node-zwave-js/#/api/controller?id=quotstatistics-updatedquot):** Provides statistics about communication between the controller and other devices, allowing you to troubleshoot your network's RF quality.
+* **Third-party data opt-in/out:** Allows you to opt-in or out of telemetry that the Z-Wave JS project collects to help inform development decisions, influence manufacturers, etc. This telemetry is disabled by default and has to be opted in to be activated.
+
+### Device panel
+
+The following features can be accessed from the device panel of a Z-Wave device:
+
+![Z-Wave device panel](/images/integrations/z-wave/z-wave-device-info.png)
+
+* **Configure:** Provides an easy way to look up and update configuration parameters for the device. While there is an existing service for setting configuration parameter values, this UI may sometimes be quicker to use for one-off changes.
+* **Re-interview:** Forces the device to go through the interview process again so that Z-Wave-JS can discover all of its capabilities. Can be helpful if you don't see all the expected entities for your device.
+* **Heal:** Forces the device to rediscover its optimal route back to the controller. Use this if you think you are experiencing unexpected delays or RF issues with your device. Your device may be less responsive during this process.
+* **Remove failed:** Forces the controller to remove the device from the controller. Can be used when a device has failed and it can't go through the normal exclusion process.
+* **[Statistics](https://zwave-js.github.io/node-zwave-js/#/api/node?id=quotstatistics-updatedquot):** Provides statistics about communication between this device and the controller, allowing you to troubleshoot RF issues with the device.
+* **Update:** Updates a device's firmware using a manually uploaded firmware file. Only some devices support this feature (controllers and devices with the Firmware Update Metadata Command Class).
+* **Download diagnostics:** Exports a JSON file describing all the entities and their properties.
```

Comment:
Should we link that to: <https://www.home-assistant.io/docs/configuration/troubleshooting/#download-diagnostics>?

Reply:
Depending on the integration, it can provide diagnostics for a specific device or the whole config entry.

In case an integration doesn't support it for a specific device, the config entry is offered when device diagnostics are downloaded as a fallback.

In the case of Z-Wave, both device and full config entry are available.